projects/angular-sdk/src/angular-sdk.module.ts

```diff
@@ -36,14 +33,22 @@ import { HopperEventsService } from './services/hopper-events.service';
 import { GlobalEventComponent } from './components/global-event.component';
 import { LoggerService } from './services/logger.service';
 
+import localeFr from '@angular/common/locales/fr';
+import localeEn from '@angular/common/locales/en';
+import localeZh from '@angular/common/locales/zh';
+
+registerLocaleData(localeFr, 'fr');
```

Comment:
is partner supposed to pass `fr` and `en` values? can it be` fr-ca` etc?

Reply:
AC is currently passing en/zh/fr (not regionalized). here are the errors i'm seeing in the console with the reverted commit

![Screenshot 2024-05-29 at 1 51 57 PM](https://github.com/hopper/hc-airlines-angular/assets/4723782/32278834-96bc-4688-a542-376198de0738)